css/login.css

```diff
@@ -0,0 +1,84 @@
+body{
+    box-sizing: border-box;
+    position: relative;
+}
+
+#external {
+    background-image:"../assets/feedback-banner.jpg";
+    height: 35rem;
+    position:relative;
+}
+
+/*@Alex I forgot to remove the url
+I cannot see from the code above it, but you asked about the relative and absolute so i hope it was about the footer/body relation.
```

Comment:
Also, could you split these painfully long lines onto multiple lines, pretty please? It's a pain to review them in GitHub.